org.sasview.sasview.yml

```diff
@@ -0,0 +1,59 @@
+id: org.sasview.sasview
+runtime: org.kde.Sdk
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: sasview
+finish-args:
+  - "--socket=fallback-x11"
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--filesystem=~/sasview.log"
+  - "--filesystem=~/.sasview"
+  - "--filesystem=~/.sasmodels"
+modules:
+  - shared-modules/glu/glu-9.json
+  - name: sasview
+    buildsystem: simple
+    build-commands:
+      - mkdir $FLATPAK_DEST/bin
+      - cp -r . $FLATPAK_DEST/bin/
+    sources:
+      - type: archive
+        url: https://github.com/SasView/sasview/releases/download/v6.0.0/SasView-6.0.0-Ubuntu-22.04.tar.gz
```

Comment:
which architectures does this support and is this a binary package?

Reply:
inside the archive all I see is bunch of python modules and 821 libraries:

```
find . -name "*.so*"|wc -l

821
```

this is not how you ship things. Python modules should be built as part of the manifest.

Reply:
And all of them are x86_64 while you are building for aarch64 too here too.

```
file _cffi_backend.cpython-311-x86_64-linux-gnu.so 
_cffi_backend.cpython-311-x86_64-linux-gnu.so: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=9dbd6756de135c8c7b48c480a28b99cf7b4d522d, with debug_info, not stripped
```

Reply:
For rust dependencies, you have to extract the tar.gz from pypi and run flatpak-cargo-generator on it from the same repo as pip-generator.

Search for apps on Flathub using python-cryptography.

Reply:
In that case, I'll take a look in this when I have time (likely Monday now), and see if I can get this to work.